Executables/Game/Packets/Character.cs

```diff
@@ -43,27 +43,26 @@ public class Character
         public ushort Port { get; set; }
         [Field(17)]
         public byte SkillGroup { get; set; }
+        public byte Empire { get; set; }
 
-        public static Character FromEntity(Player player)
+        public static Character FromEntity(Player player) => new Character
```

Comment:
I personally don't like this lambda like functions if it's not just a simple method forwarding. It looks like a property but it's not imo